pkg/holmes/holmes.go

```diff
@@ -0,0 +1,162 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package holmes
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"mosn.io/holmes"
+	v2 "mosn.io/mosn/pkg/config/v2"
+	"mosn.io/mosn/pkg/mosn"
+	"time"
+)
+
+var (
+	h *holmes.Holmes
+)
+
+/*
+ * We intergrate Holmes to MOSN builtin extension.
+ * See Holmes: https://github.com/mosn/holmes
+ */
+
+type holmesConfig struct {
```

Comment:
maybe this config should belong to Holmes.

Reply:
Holmes only exposes the `Withxxx` APIs, not a config struct.